build_tools/ci/cpu_comparison/run_test.sh

```diff
@@ -130,6 +122,8 @@ fi
 
 
 source $XRT_DIR/setup.sh
+# Circumvent xclbin security (no longer needed as of April 2024 XDNA driver)
```

Comment:
Is this the reason we don't need to do signing?

Reply:
Yup. This line will be removed too as soon as we can update the CI machine driver